### DISCUSSION TITLE
How to use the Jointjs's Demo-Javascript-ast-visualizer in my own vue project?

### DISCUSSION THREAD

**Discussion by zouqiBupt** [clientIO/joint#1810] [Q&A]
I have the source code of Demo-Javascript-ast-visualizer,But i don't know how to use in my own vue-project
I mean ,How can I use the source code so that I can input JS code like the DEMO in my VUE project and then output the visualized results of the AST?
![image](https://user-images.githubusercontent.com/24296616/190886443-f35779f6-2565-46a5-8590-b891021facdc.png)
I use iframe to set html in vue but seems to have some bug with jquery and other dependency
I install the dependency by follow -->https://github.com/clientIO/joint-plus-tutorial-vue
Using the License of Open Source License
![image](https://user-images.githubusercontent.com/24296616/190886666-74fe6a03-74f2-46e4-989f-a78a7504a878.png)
![image](https://user-images.githubusercontent.com/24296616/190886670-c1574931-9bc3-4795-a4fc-b8c33ee28089.png)
My response of rappid.js and rappid.css seems not correct

**Comment by kumilingus** [comment]
> Using the License of Open Source License

`rappid.js` is part of our commercial package `jointjs+`. 

> Uncaught SyntaxError : Unexpected token '<'

I don't think this is related to `jointjs`. Perhaps you'll find answers [here](https://stackoverflow.com/questions/18561556/syntax-error-unexpected-token).
